### PR TITLE
Cleanup

### LINKED ISSUE #22: Remove PACTS
**Description**
After #21 we no longer seem to need pacts.

We should remove them.

### PR DESCRIPTION
**Pull request by Masclins**
**Description**

Some cleanup, removing the pacts and unused stuff.

**Issues Resolved**

Resolves #22 

**Details**

I added some configuration to Eastwood so it finds unused arguments, ns and so on.
Also removed everything it found not to be used.

Also, removed pacts and all relating to it, according to #22 

**Hotspots**

`api.handler` had to be removed from eastwood's Travis calling because `:unused-fn-args` detects the entire ns as unused (or so it seemed to me). I considered better to keep this configuration and not include `api.handler` since the handler is not likely to change a lot.
This was the warning that would make Travis fail
```
== Linting api.handler ==
src/api/handler.clj:1:1: unused-fn-args: Function arg request__14364__auto__ never used
```

Still eastwood will be able to be called locally by `lein eastwood` showing some other warnings that might appear on `api.handler`.

It would be nice to find a way to ignore that single warning.

**Checklist for contribution requirements**

- [x] The tests pass
- [ ] The code has tests
- [ ] The tests are well-designed
- [ ] The code is readable
- [ ] The code is well-organized
- [ ] Any rule changed or included is also on the Manual

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,10 +2,7 @@ language: clojure
 script:
     - lein test
     - lein yagni
-    - lein eastwood
+    - lein eastwood '{:exclude-namespaces [api.handler]}'
```
Comment (kenan-rhoton) [inline .travis.yml:5]: if we have to execute eastwood with this options, it's better to have it defined in the `project.clj`

File: project.clj [hunk 2]
```diff
@@ -19,34 +19,8 @@
             [venantius/yagni "0.1.7"]]
   :cloverage {:ns-exclude-regex [#"api\.handler"]
```
Comment (kenan-rhoton) [inline project.clj]: I think at this point, since `api.handler` is now testable, we should not exclude it from cloverage... and test it properly!

**Review by kenan-rhoton — CHANGES_REQUESTED** [review]
Thanks for the cleanup! Can you change the eastwood options and give your view on the api.handler testing?

File: .travis.yml (hunk 1 above)
Comment (Masclins) [reply .travis.yml:5]: I prefere it this way. I think's it's worth that when using `lein eastwood` from our machines we get all warnings from `api.handler`.

Ideally I'd exclude only the warning saying that the api.handler ns is not called, but I don't know how.

File: project.clj (hunk 2 above)
Comment (Masclins) [reply project.clj]: Gonna see if it's possible.

**Comment by Masclins** [comment]
Very important...

This seems to break `lein test-refresh`, but `lein test` and the first set of tests of `lein test-refresh` work well.

I think it has to do with not doing a new mock-request when refreshing the tests.

@kenan-rhoton if not willing to merge until test-refresh is solved, I'll need your help.

**Comment by Masclins** [comment]
Ignore last message, test-refresh has been solved with latest commit.

**Review by kenan-rhoton — APPROVED** [review]

**Pull request merged**